src/media/css/account.styl

```diff
@@ -81,6 +81,10 @@
         display: none;
     }
 }
+.html-rtl .region span {
```

Comment:
We could also use a block-level mixin to communicate differences if you want to keep them in the same file. I find this pattern exceptionally elegant, but I know @ngokevin isn't a fan.

    rtl() {
      .html-rtl & {
        {block}
      }
    }

    .region span {
        background-position: 0 1px;
        +rtl() {
            background-position: 100% 1px;
        }
    }